apps/src/templates/studioHomepages/RecentCourses.jsx

```diff
@@ -17,50 +18,54 @@ const RecentCourses = React.createClass({
 
   render() {
     const { courses, showAllCoursesLink, isTeacher, heading, isRtl } = this.props;
+    const topFourCourses = courses.length > 4 ? courses.slice(0,4) : courses;
```

Comment:
Is 4 a magic number that should be extracted to a named constant along with a comment about what it means?